pkg/controller/service/controller.go

```diff
@@ -311,6 +312,10 @@ func (s *Controller) syncLoadBalancerIfNeeded(service *v1.Service, key string) (
 	var err error
 
 	if !wantsLoadBalancer(service) || needsCleanup(service) {
+		if v1helper.LoadBalancerStatusEqual(previousStatus, &v1.LoadBalancerStatus{}) {
```

Comment:
Drop this carry commit, link to https://github.com/kubernetes/kubernetes/pull/63926#issuecomment-585545637 and make BZ for @ironcladlou to verify.

Reply:
Done. bz: https://bugzilla.redhat.com/show_bug.cgi?id=1814044